hydra-node/json-schemas/api.yaml

```diff
@@ -223,23 +223,6 @@ components:
             type: string
             enum: ["Abort"]
 
-    Commit:
```

Comment:
Must: remove the entry for this in the `clientInput` operation [above](https://github.com/input-output-hk/hydra/blob/f0655942bde4148ff59b79bb399e9c00b17cda8c/hydra-node/json-schemas/api.yaml#L105)

Should: update the state diagram in the [docs](https://github.com/input-output-hk/hydra/tree/f0655942bde4148ff59b79bb399e9c00b17cda8c/docs/core-concepts/behavior.md) to not have a `Commit` event/client input, but instead mention that a commit transaction needs to be built + submitted (maybe even refer `POST /commit`)